src/main/java/sirius/biz/importer/txn/ImportTransactionHelper.java

```diff
@@ -72,8 +72,10 @@ public ImportTransactionHelper start() {
      * @return the helper itself for fluent method calls
      */
     public ImportTransactionHelper start(String source) {
-        this.transactionId = System.currentTimeMillis();
-        this.source = source;
+        start();
+        if (Strings.isFilled(source)) {
+            this.source = source.trim();
```

Comment:
What if the `source` is empty after trimming? Do we consider both, `null` and the empty string further down the line?

Reply:
Yes, we do. The value is only used if Strings.isFilled delivers something.